ext/zip/php_zip.stub.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/** @return resource|int|false */
+function zip_open(string $filename) {}
+
+/**
+ * @param resource $zip
+ */
+function zip_close($zip): void {}
+
+/**
+ * @param resource $zip
+ * @return resource|false
+ */
+function zip_read($zip) {}
+
+/**
+ * @param resource $zip_dp
+ * @param resource $zip_entry
+ */
+function zip_entry_open($zip_dp, $zip_entry, string $mode = null): bool {}
```

Comment:
`null` isn't a valid value for this argument. It's somewhat weird though, because it's unused in the implementation. I guess we can just do `string $mode = ""` here.

Reply:
Assuming the [documention](https://www.php.net/manual/en/function.zip-entry-open.php) is correct (what it likely isn't), `string $mode = "rb"` would be sensible.

Reply:
any consensus on this, between `''` and `'rb'` ?

Reply:
Any value would be okay here ... if the docs say `"rb"` then let's go with that.